scripts/publish-docker

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import os.path
+import subprocess
+import sys
+
+
+def main():
+
+    sys.stdout.flush()
+
+    cmd = ["docker", "login"]
+    login = subprocess.check_output(["gopass", "gs/ci/dockerhub/username"]).decode()
+    password = subprocess.check_output(["gopass", "gs/ci/dockerhub/password"])
+    prefix = ""
+    cmd += ["--username=" + login, "--password-stdin"]
+    process = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    output, output_err = process.communicate(input=password)
+    if process.returncode != 0:
+        if output:
+            print(output.decode())
+        if output_err:
+            print(output_err.decode())
+        sys.exit(1)
+
+    image = "camptocamp/lyrx2sld:latest"
```

Comment:
the image name/tag should be arguments of the script, so that we can push several images (esp. tags) if needed easily directly from the workflow when calling the script